catroid/src/main/res/values/strings.xml

```diff
@@ -81,6 +81,16 @@
         You may also report the project on its page on the Catrobat community platform in case
         you are sure that it is harmful.</string>
 
+    <!-- Replace Api dialog -->
+    <string name="api_replacement_dialog_neutral">Upload with Keys anyway</string>
+    <string name="api_replacement_dialog_accept">Upload with Keys replaced</string>
+    <string name="api_replacement_dialog_warning">The project contains bricks that access the
+        web, and also seems to contain API keys. Please see our &lt;a href=%1s&gt;wiki&lt;/a&gt; why
```

Comment:
```suggestion
        web, and also seems to contain API keys. Please see %1s why
```
This shall be a clickable link, and the link shall be visible. It must be easy for translators.